skfda/tests/test_irregular_operations.py

```diff
@@ -0,0 +1,847 @@
+"""Test the operations of the FDataIrregular structure."""
+from typing import Optional, Tuple
+
+import numpy as np
+import pytest
+
+from skfda.representation import FDataGrid, FDataIrregular
+from skfda.representation.basis import (
+    Basis,
+    BSplineBasis,
+    FDataBasis,
+    FourierBasis,
+    TensorBasis,
+)
+
+from ..typing._numpy import Any, ArrayLike
+
+############
+# MACROS
+############
+SEED = 2906198114
+
+NUM_CURVES = 100
+MAX_VALUES_PER_CURVE = 10
+DIMENSIONS = 2
+N_BASIS = 5
+DECIMALS = 4
+
+random_state = np.random.RandomState(seed=SEED)
```

Comment:
Maybe we should use [`default_rng`](https://numpy.org/doc/stable/reference/random/generator.html) instead if possible, which uses a `Generator`, the modern alternative to `RandomState`.